src/pages/signin/SignInPageLayout/SignInPageContent.js

```diff
@@ -28,47 +26,42 @@ const propTypes = {
 
     ...withLocalizePropTypes,
     ...windowDimensionsPropTypes,
+    ...withKeyboardStatePropTypes,
 };
 
-const SignInPageContent = props => (
-    <ScrollView
-        keyboardShouldPersistTaps="handled"
-        showsVerticalScrollIndicator={false}
-        style={[
-            styles.h100,
-            !props.isSmallScreenWidth && styles.alignSelfCenter,
-            !props.isSmallScreenWidth && styles.signInPageWideLeftContainer,
-        ]}
-        contentContainerStyle={[
-            scrollViewContentContainerStyles,
-            styles.alignItemsCenter,
-            !props.isSmallScreenWidth && styles.ph6,
-        ]}
-    >
-        <View style={[styles.flex1, styles.flexRow]}>
-            <View style={[
-                styles.flex1,
-                styles.signInPageNarrowContentContainer,
-            ]}
-            >
-                <SignInPageForm style={[
+const SignInPageContent = (props) => {
+    const dismissKeyboardWhenTappedOutsideOfInput = () => {
+        // This prop comes from with
+        if (!props.isShown) {
+            return;
+        }
+        Keyboard.dismiss();
+    };
+
+    return (
+        <TouchableWithoutFeedback onPress={dismissKeyboardWhenTappedOutsideOfInput}>
+            <View
+                style={[
                     styles.flex1,
-                    styles.alignSelfStretch,
-                    props.isSmallScreenWidth ? styles.ph5 : styles.ph4,
+                    styles.signInPageLeftContainer,
+
+                    // Restrict the width if the left container only for large screens. For smaller screens, the width needs to be fluid to span the entire width of the page.
+                    !props.isMediumScreenWidth && !props.isSmallScreenWidth && styles.signInPageWideLeftContainer,
```

Comment:
NAB: Wondering why we don't have a `isLargeScreenWidth` prop for times like these

Reply:
Actually, that's a great point. I want to refactor it a little to have this.